samples/app-dev/Connectors/Miscellaneous/ODataS4hana/readme.md

```diff
@@ -0,0 +1,253 @@
+# OData with SAP S4 Hana Example
+
+
+## Description
+
+This example demonstrates how we can create SAP S4 Hana connection using OData connector and use all OData CRUD activities for SAP S4 Hana server.
+OData app with SAP S4 Hana bascially contain 2 activities i.e. OData Query and OData Modify. OData Modify contains three operation i.e. Create, Update and Delete. 
+The main purpose of these activities are:
+a) To fetch SAP S4 Hana data using OData Query activity.
+b) To create data in SAP S4 Hana server using Create operation of OData Modify activity.
+c) To update data in SAP S4 Hana server using Update operation of OData Modify activity.
+d) To delete data in SAP S4 Hana server using Delete operation of OData Modify activity.
+
+## Prerequisites
+
+1. Ensure that SAP S4 Hana services are up and running
```

Comment:
Please add one more pre-requisite
Make sure that user have correct Root URL of OData service (i.e., SAP S4 HANA)